sdk/release/artifacts.yaml

```diff
@@ -189,7 +189,7 @@
   type: jar-scala
 - target: //ledger-service/cli-opts:cli-opts
   type: jar-scala
-- target: //libs-scala/jwt:jwt
```

Comment:
we publish `//canton:daml-common-staging_daml-jwt` instead of `//libs-scala/jwt:jwt`